deploy/addons/dashboard/dashboard-dp.template

```diff
@@ -37,7 +37,7 @@ spec:
     spec:
       containers:
       - name: kubernetes-dashboard
-        image: {{default "k8s.gcr.io" .ImageRepository}}/kubernetes-dashboard-amd64:v1.10.1
+        image: {{default "k8s.gcr.io" .ImageRepository}}/kubernetes-dashboard-IMAGETAG:v1.10.1
```

Comment:
The term "tag" is already used for what comes after the colon (v1.10.1).

What do you think about something like ARCH or PLATFORM?

Reply:
changes to ARCHTAG.